backend/lightning/lightning.go

```diff
@@ -108,14 +108,18 @@ func (lightning *Lightning) Activate() error {
 		return err
 	}
 
-	go lightning.connect(true)
+	if err = lightning.connect(true); err != nil {
+		return err
+	}
 
 	return nil
 }
 
 // Connect needs to be called before any requests are made.
 func (lightning *Lightning) Connect() {
```

Comment:
I see this is called on app launch in backend.Start(). Isn't this overkill as many will not have an active LN account?

Reply:
Also, if it takes a while, this would block/delay the app launch, right?

Reply:
It should be really fast, as the node has been already started. Anyway, here the `go ...` doesn't hurt. I'll do it :+1: